DREAM3DReviewFilters/ImportQMMeltpoolH5File.cpp

```diff
@@ -165,16 +166,87 @@ void ImportQMMeltpoolH5File::initialize()
   setCancel(false);
 }
 
+// -----------------------------------------------------------------------------
+std::pair<int32_t, std::string> checkFile(const std::string& filePath)
```

Comment:
This function should be moved to the anonymous namespace.

Reply:
Done